apps/browser/qml/pages/components/SecondaryBar.qml

```diff
@@ -74,6 +74,21 @@ Column {
         }
     }
 
+    TextSwitch {
+        height: overlay.toolBar.rowHeight
+        enabled: webView.contentItem
+        automaticCheck: false
+
+        //: Label for text switch that reloads page in desktop mode
+        //% "Desktop site"
```

Comment:
This introduces a text alignment issue but good starting point.

We should have icons (mobile & desktop), text changing accordingly (Show mobile site, Show desktop site), and use the OverlayListItem to get the alignment fixed. Filed JB#53202. Please fix that one after this has been landed.